core/vm/gas_table.go

```diff
@@ -188,8 +189,9 @@ func gasSStoreEIP2200(evm *EVM, contract *Contract, stack *Stack, mem *Memory, m
 	}
 	// Gas sentry honoured, do the actual gas calculation based on the stored value
 	var (
-		y, x         = stack.Back(1), stack.Back(0)
-		current, err = evm.StateDB.GetState(contract.Address(), x.Bytes32())
+		y, x                      = stack.Back(1), stack.Back(0)
+		internalContractAddr, err = contract.Address().InternalAddress()
+		current, _                = evm.StateDB.GetState(*internalContractAddr, x.Bytes32())
```

Comment:
no need of the error catch?

Reply:
https://github.com/dominant-strategies/go-quai/blob/geth-fork/address-interface/core/vm/gas_table.go#L104